### ISSUE TITLE
is there any version for ARM platform

### ISSUE THREAD

**Issue by zhanb** [Corvusoft/restbed#397]
hi,I want to compile restbed on ARM not x86 platform ,what can I do?

**Comment by ben-crowhurst** [comment]
Currently we don't support ARM.  Reach out to support@corvusoft.co.uk if you would like this implemented.

**Comment by ben-crowhurst** [comment]
Let me rephrase my previous comment.  We support ARM in as much as the C++ compiler within the operating system supports C++11.

If you have a C++11 compliance compiler you will be able to run Restbed on ARM platforms.
